.gitignore

```diff
@@ -1,3 +1,3 @@
-/node_modules
 /.idea
-/dist/
\ No newline at end of file
+node_modules
+dist
```

Comment:
add 
src/**/*.js
src/**/*.map

Reply:
Added outDir to tsconfig, I think, that adding of this lines not need already.